azext_iot/tests/dps/device_registration/test_iot_device_registration_individual_int.py

```diff
@@ -0,0 +1,541 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+from azext_iot.common.shared import EntityStatusType, AttestationType
+from azext_iot.tests.dps import (
+    DATAPLANE_AUTH_TYPES,
+    CERT_NAME,
+    CERT_PATH,
+    KEY_PATH,
+    SECONDARY_CERT_NAME,
+    SECONDARY_CERT_PATH,
+    SECONDARY_KEY_PATH,
+    IoTDPSLiveScenarioTest
+)
+from azext_iot.tests.dps.device_registration import compare_registrations
+
+
+class TestDPSDeviceRegistrationsIndividual(IoTDPSLiveScenarioTest):
+    def __init__(self, test_case):
+        super(TestDPSDeviceRegistrationsIndividual, self).__init__(test_case, cert_only=False)
+        self.id_scope = self.get_dps_id_scope()
+        self.add_hub_permissions()
```

Comment:
self.add_hub_permissions() on the __init__ and below test?

Reply:
fml I thought I got rid of that

Reply:
got rid of the entire command